_posts/2019-06-3-4-contribute-open-source.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Contributing to open source"
+date: 2019-06-03 09:00:00
+categories: ["open-source"]
+author: "bronzdoc"
+---
+
+Contributing to open source might be scary, you might be thinking that you PR will not have a good quality or that people will judge you by your code, or that fixing that little typo is not worth it. I had all this questions in my head before i did my first contribution to open source and stoped me for a long time to contribute.
+
+So my suggestions is, start small, look for low hanging issues in a project you want to contribute, start looking at the issues and see if they have been solved previously and suggest a fix, by doing this you will start learning the code base and it will give you confidence to start tackling more complex issues.
+
+This is a set of steps that would help you get started in your open source journey:
+
+### If you think you found a bug:
+1. Search in Stack Overflow
+2. Search in GitHub's Issues
+3. If I don't find anything anywhere, submit an issue
+4. If I know a patch that works, submit a PR
+5. Wait for feedback
+
+### If you think something will be a nice feature:
+1. Search in GitHub's Issues if that feature is already proposed
+2. If you don't find anything in the issue tracker, submit an issue explaining why you think your feature will add value to the project
+3. Wait for feeback
+4. if you got a positive feedback about your feature, submit a PR
+5. Wait for feddback
+
+
+These steps are not written in gold, but i think they might help you get you started.
```

Comment:
@bronzdoc I think you can say these are the steps that we follow at Ombu Labs.

Reply:
@etagwerker something like the following? 
`These steps are not written in gold, but it is what we use at Ombulabs and we think they might help you get you started.`